servicetalk-log4j2-mdc-utils/build.gradle

```diff
@@ -19,7 +19,7 @@ apply plugin: "io.servicetalk.servicetalk-gradle-plugin-internal-library"
 dependencies {
   testImplementation enforcedPlatform("org.junit:junit-bom:$junit5Version")
 
-  api "org.apache.logging.log4j:log4j-api:$log4jVersion"
+  api "org.apache.logging.log4j:log4j-core:$log4jVersion"
```

Comment:
Why is this reverted back to `log4j-core`?

Reply:
let's continue here: https://github.com/apple/servicetalk/pull/2217#discussion_r874240630